src/plugins/mf_priority.cpp

```diff
@@ -108,6 +108,13 @@ int64_t priority_calculation (flux_plugin_t *p,
     fshare_factor = b->fairshare;
     queue_factor = b->queue_factor;
 
+    priority = round ((fshare_weight * fshare_factor) +
+                      (queue_weight * queue_factor) +
+                      (urgency - 16));
+
+    if (fshare_factor < 0.00000)
```

Comment:
Also, I don't _think_ `0.00000` is any different than `0.0` or even `0.`.

Reply:
Yeah, in fact maybe I could use some clarification here too. I always thought that a double value of `0.0001` was treated as larger than `0.00000`. The flux-accounting DB stores and the bulk update script sends fairshare values to the 5th decimal place, so that's why I included the test for `< 0.00000`, and my assumption was that a user/bank combo who had a fairshare value of `0.00001` or larger would be allowed to run jobs (although that is really really low). But now that I'm typing this out, I'm thinking it would be better to instead just compare to exactly `0`, as you suggested above? As that's what the test should really be looking for, for a value of exactly `0`.

Reply:
Yes, `0.001` is larger than `0.00000`, but like I said, `0.00000` and `0.0` (and `0.`) are all the same as far as I know.

Note that I think you have to be careful with calculated floating point numbers due to rounding errors, It is very difficult to get an "exact" number with floating point calculations (and I didn't see if the fairshare value is rounded). So you could calculate a fairshare value that is _very_ close to zero (e.g. `1.0e-10`) and which it may be convenient to treat as a literal zero. However, I'm not sure that a real problem here since even `1.0e-10` is greater than zero. Just thought I'd give something to think about (I'm not sure if there is potential for a rounding error to cause a fairshare value to be a very very small negative number which should instead be treated as `0`.)

Reply:
Yeah, I think I see what you're saying. Thanks for this note. I don't _think_ it should be an issue since fairshare values are not rounded in the priority calculation, and only the final priority is rounded. So I think I'll push a fix to just compare `fshare` to `0`.